modules/mongodb/mongodb.go

```diff
@@ -79,6 +79,29 @@ func WithPassword(password string) testcontainers.CustomizeRequestOption {
 	}
 }
 
+// WithReplicaSet configures the container to run a single-node MongoDB replica set named "rs".
+// It will wait until the replica set is ready.
+func WithReplicaSet() testcontainers.CustomizeRequestOption {
+	return func(req *testcontainers.GenericContainerRequest) error {
+		req.Cmd = append(req.Cmd, "--replSet", "rs")
```

Comment:
One last question: do we want users to provide their replicasets? Or is it just fine to keep it constant, as `rs`?

Reply:
According to the [official documentation](https://www.mongodb.com/docs/manual/reference/configuration-options/#mongodb-setting-replication.replSetName), the name serves to define the replica set that the mongo belongs. Since this will be a single-node replica set, I believe it is simpler to use a constant name.